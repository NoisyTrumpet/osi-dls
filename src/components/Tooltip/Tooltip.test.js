import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Tooltip } from './';

describe('<Tooltip />', () => {
  test('should renders without props ', () => {
    const wrapper = shallow(<Tooltip />);
    expect(wrapper.getElement()).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
