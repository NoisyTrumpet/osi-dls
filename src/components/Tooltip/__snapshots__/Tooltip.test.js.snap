// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tooltip /> should renders without props  1`] = `
<div
  className="tooltip-wrapper"
>
  <button
    className="tooltip tooltip-trigger center-align"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    tabIndex="0"
    type="button"
  >
    <Typography
      variant="body-medium"
    />
    <div
      className="tooltip-content"
    >
      <div
        aria-hidden={true}
        className="tooltip-overlay bottom center-align text-align-left"
        data-animate="modal"
        data-open={false}
        data-type="modal"
        role="tooltip"
        tabIndex="-1"
      >
        <Typography
          variant="helper"
        />
        <i
          className="caret"
        />
      </div>
    </div>
  </button>
</div>
`;
